cmd/rancher_release/set_kdm_branch_references.go

```diff
@@ -3,11 +3,18 @@ package main
 import (
 	"context"
 	"errors"
+	"os"
+	"strconv"
+	"strings"
 
+	"github.com/rancher/ecm-distro-tools/exec"
 	"github.com/rancher/ecm-distro-tools/release/rancher"
+	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli/v2"
 )
 
+var err error
```

Comment:
I can imagine some scenarios where a package-level err could lead to problems.

Reply:
Yeah, probably safer to have it in function scope